.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug Report
+about: Create a bug report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Steps To Reproduce**
+Please be specific!
+1. Go to '...'
+2. Click on '....'
+3. etc.
+
+**Sample Code**
+If applicable, please include Sample code to reproduce the issue.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Actual Behavior**
+A clear and concise description of what actually happens.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Device Information (please complete the following information):**
+ - OS: [e.g., WSL2 on Win10, Mac]
+ - Deployment: [e.g., Docker-version, docker-compose, Python application]
+ - Browser: [e.g., chrome, safari]
+ - Version: [e.g., 0.2.0-dev]
+
+**Additional context**
+Add any other context about the problem here. If applicable, please include why you think the bug is occurring and/or troubleshooting you have already performed.
```

Comment:
These descriptions can be placed in comments, to hide them from the finished issue, but still be helpful for the creator.
```suggestion
**Describe the bug**
<!-- A clear and concise description of what the bug is. -->

**Steps To Reproduce**
Please be specific!
1. Go to '...'
2. Click on '....'
3. etc.

**Sample Code**
<!-- If applicable, please include Sample code to reproduce the issue. -->

**Expected behavior**
<!-- A clear and concise description of what you expected to happen. -->

**Actual Behavior**
<!-- A clear and concise description of what actually happens. -->

**Screenshots**
<!-- If applicable, add screenshots to help explain your problem. -->

**Device Information (please complete the following information):**
 - OS: [e.g., WSL2 on Win10, Mac]
 - Deployment: [e.g., Docker-version, docker-compose, Python application]
 - Browser: [e.g., chrome, safari]
 - Version: [e.g., 0.2.0-dev]

**Additional context**
<!-- Add any other context about the problem here. If applicable, please include why you think the bug is occurring and/or troubleshooting you have already performed. -->
```